contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -39,6 +40,9 @@ def __init__(
       this value to True.
     :param style_ignore_path: relative path to file specifying patterns of files to ignore. The syntax
       supported is the same as the .eslintignore/.gitignore syntax.
+    :param bin_executables: A map of executable names to local file name. If a single executable is
+                            specified (a string), the package name will be the executable name
+                            and the value will be the local file name per package.json rules.
```

Comment:
It wasn't clear to me on first read that this could be a string or dict. Maybe add a `:type bin_executables:` bit?

Maybe something like would be clearer? I may be missing something conceptually though

    Executables for this node target. Specify one executable with a string,
    or a multiple with a dict mapping executable names to files. Supplying
    a string will set the package name as the executable name and the file
    will be found per package.json rules.


I'd also like to see test cases for both types of bin_executable input types.

Reply:
done